src/components/Error/style.css

```diff
@@ -62,9 +62,7 @@
   position: absolute;
   height: 100%;
   width: @icon-width;
-  @media (--small-viewport) {
-    width: @icon-width-small;
-  }
+  width: @icon-width-small;
```

Comment:
Should the above line be removed?

Reply:
Oh, I missed the line above. I've refactored it. Thanks